src/config/mod.rs

```diff
@@ -87,6 +90,42 @@ impl PersistentData for Config {
     }
 }
 
+#[derive(Deserialize, Serialize, Debug, PartialEq, Clone)]
+pub enum ColumnConfig {
+    V1 {
+        local_version_width: u16,
+        remote_version_width: u16,
+        status_width: u16,
+    },
+}
```

Comment:
I've added this as an enum so we don't break deserialization in the future if this needs to change (such as adding ordering & hidden).

Reply:
Also, "title" width isn't defined here since I always leave "title" as `Length::Fill". This ensures the table still resizes nicely when the application does.

Reply:
Make sense, good thinking.